CHANGELOG.md

```diff
@@ -1,6 +1,7 @@
 ## v0.38.0 (October 24, 2022)
 
 FEATURES:
+* r/tfe_organization: Add `allow_force_delete_workspaces` attribute to set whether admins are permitted to delete workspaces with resource under management. ([#661](https://github.com/hashicorp/terraform-provider-tfe/pull/661))
```

Comment:
This needs to be moved outside of the 0.38.0 section, since that has been released now

Reply:
ah good catch ! 🙇